metals/src/main/scala/scala/meta/internal/decorations/SemanticdbTreePrinter.scala

```diff
@@ -48,20 +56,27 @@ class SemanticdbTreePrinter(
           .reduceLeft((x, y) => s"$x $y")
         s"$mapped ${printType(tp)}"
       // this should not need to be printed but just in case we revert to semanticdb printer
-      case s.UniversalType(_, tpe) =>
+      case s.UniversalType(scope, tpe) =>
         if (isHover)
           Print.tpe(Format.Detailed, t, symtab)
-        else s"[ ... ] => ${printType(tpe)}"
+        else s"[${printScope(scope)}] => ${printType(tpe)}"
       case s.ExistentialType(tpe, _) =>
         if (isHover)
           Print.tpe(Format.Detailed, t, symtab)
-        else s"${printType(tpe)} forSome { ... }"
-      case s.StructuralType(tpe, _) =>
-        if (isHover)
+        else
+          s"${printType(tpe)}"
+      case s.StructuralType(tpe, scope) =>
+        if (isHover) {
           Print.tpe(Format.Detailed, t, symtab)
-        else s" ${printType(tpe)} { ... }"
+        } else {
+          s"${printType(tpe)} {${printScope(scope)}}"
+        }
     }
 
+  def printScope(scope: Option[s.Scope]): String = {
+    if (scope.exists(_.hardlinks.nonEmpty)) "..." else ""
```

Comment:
Just a nit, but did you want a `...` or `…` here for this?

Reply:
I actually intended to change it :sweat_smile:  Changed to using ellpsis when applicable.